src/client/pages/FamilyGroup.css

```diff
@@ -1,29 +1,41 @@
 #familyGroup{
```

Comment:
nit: This id isn't necessary to namespace these styles for just the family group page. The CSS Modules will handle that. This could be removed as well as the `id` on line 35 of `FamilyGroup.jsx`